frontend/src/AppRoutes/pageComponents.ts

```diff
@@ -100,6 +100,10 @@ export const MySettings = Loadable(
 	() => import(/* webpackChunkName: "All MySettings" */ 'pages/MySettings'),
 );
 
+export const Logs = Loadable(
+	() => import(/* webpackChunkName: "SomethingWentWrong" */ 'pages/Logs'),
```

Comment:
Logs

Reply:
Thanks for highlighting this. Though this PR is not ready to merge

Reply:
yeah 😛